assets/resources/scss/_navMenu.scss

```diff
@@ -28,11 +28,112 @@ header {
   &__list {
     display: flex;
     flex-direction: column;
+    position: fixed;
     width: 100%;
+    max-width: var(--max-content-width);
+
+    background-color: var(--primary-color);
+    z-index: 1;
+
+    &--hide-vertical {
+      transition: all 450ms ease-in-out 0ms;
```

Comment:
If you are putting the delay 0ms maybe you coud not put it at all?

Reply:
Changed.